### DISCUSSION TITLE
To-Do list for 2.0 release

### DISCUSSION THREAD

**Discussion by tux-linux** [Quill-OS/quill#36] [General]
- [x] X11 freeze fix + onboard
- [x] Rewrite init
- [x] Recompile platform plugin with fixes for user apps in rootfs
- [x] Folders in localLibraryWidget (@Szybet)
- Reader: fit to screen size better (@Szybet's work for 2.1)
- [x] Fix built-in apps (add exit buttons)
- [x] Implement rust version of split.py and remove Python from GUI bundle
- [x] Fix FBDepth in IPD - Doesn't seem to always work but doesn't seem to be fixable either
- [x] Increase check box size in stylesheets (rootfs)
- [x] Replace UI monospace font with Roboto Mono
- [x] Qt keyboard: more special characters
- Fix aspect ratio in Reader (@Szybet's work for 2.1)
- [x] IPD: Libra: Fix LED -- WARNING: Kernel bug? LED powers on on plug-in but turns off itself after a minute or so
- [x] Add more font sizes + line spacing in Reader
- [x] Fix Wi-Fi (DHCP lease acquisition) for Glo HD
- [x] Redesign update system
- [x] Replace sketch app
- [x] Always enable UI scaling and set it to 1
- Implement audio support for KT (To-do for 2.1)
- [x] Have lockscreen work in a bare-minimum way with IPD
- [x] Add IPD_DEBUG option to Diagnostics
- [x] update @Szybet OS and see if cpu bug is still there

**Comment by Szybet** [comment]
- [x] Settings: Replace all spin boxes with something bigger
- [ ] Settings: Remove network related things
- [ ] Home page: Remove items that arent in the database anymore ( moved or removed )
- [x] Network information dialog: Remove ugly border on the left and top
- [x] Update checking: Launch it in another thread to avoid freezing the gui, an option to disable the popup
- [x] Option to disable USB dialog poping up
